cli/.eslintrc

```diff
@@ -3,28 +3,18 @@
   "env": {
     "es6": true
   },
-  "plugins": [
-    "flowtype",
-    "prettier"
-  ],
-  "extends": [
-    "plugin:flowtype/recommended"
-  ],
-  "globals": {
-  },
+  "plugins": ["flowtype"],
+  "extends": ["plugin:flowtype/recommended", "plugin:prettier/recommended"],
+  "globals": {},
   "rules": {
-    "prettier/prettier": [
```

Comment:
So basically this disable any ESLint linting about style?

Reply:
`plugin:prettier/recommended` sets the `prettier/prettier` rule to "error"